### ISSUE TITLE
Error on dashboard app when accessing an app database

### ISSUE THREAD

**Issue by rnjv** [web2py/py4web#304]
This is the error message I get:

```
  File "apps\_dashboard\__init__.py", line 140, in dbadmin
    return dict(languages=dumps(T.local.language))
  File "src\\gevent\\local.py", line 408, in gevent._gevent_clocal.local.__getattribute__
AttributeError: 'gevent._gevent_clocal.local' object has no attribute 'language'

```

**Comment by mdipierro** [comment]
Please open a ticket. I will try fix asap.

**Comment by mdipierro** [comment]
I cannot reproduce but I posted a fix. Could you please check it in master?

**Comment by nicozanf** [comment]
It's the same error of #281 , which is on Windows with gevent. It does not happen with Tornado.

What's your Operating System and the Web server used?

**Comment by nicozanf** [comment]
I've closed the other issue as duplicate.
The commit 5a838ed19ccccf27c8d8e6a1ab532343f5c929bd seems to have mitigated the effects (now we have traceback on some db only, and py4web remains running).
I have reproduced the problem with Windows (changing core.py line 1231 to `params["server"] = "gevent"`) and with Ubuntu (changing line 1236 accordingly). Then open the Dashboard, goto the Example APP, try to open all the dbs: you get errors only opening db.person and db.product, see the attachment:

[py4web.examples_Db_errors.txt](https://github.com/web2py/py4web/files/5324439/py4web.examples_Db_errors.txt)
